<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProbeSettingSlctvtBscPage</class>
 <widget class="QWizardPage" name="ProbeSettingSlctvtBscPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>499</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Basic Probe Selectivity Settings</string>
  </property>
  <property name="title">
   <string>Basic Probe Selectivity Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QLabel" name="label_1">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>This window allows selection of the probe selectivity settings.

The following options depict the selectivity settings for the middle part of the probes.
Theoretically, highly selective probes should be obtained when Target settings are set to High and Non-target settings are set to Low. If no primer-probe sets are produced using these settings, consider re-running the program after lowering the target settings and/or increasing the non-target settings.  This problem will be most noticeable when designing primer-probe sets for highly conserved sequences such as rRNA genes.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="PRISE.qrc">:/new/prefix1/images/probe_matches.jpg</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>10000</width>
         <height>10000</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Selectivity settings for target sequences</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Continuous</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_ContMatchesTarget">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string> matches at the center to bind to target sequences</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QSlider" name="horizontalSlider_target">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>15</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>4</number>
          </property>
          <property name="sliderPosition">
           <number>4</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Target</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Low</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>High</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>10000</width>
         <height>10000</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Selectivity settings for non-target sequences</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Continuous</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_ContMatchesNontarget">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string> matches at the center to bind to non-target sequences</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Non-target</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QSlider" name="horizontalSlider_nontarget">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>15</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="sliderPosition">
           <number>4</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Low</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>High</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>10000</width>
       <height>10000</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Select from the following choices</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_default">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Use default mismatch settings (and start designing probes now)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton_showAdv">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Show/change mismatch settings</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="PRISE.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>radioButton_default</sender>
   <signal>clicked()</signal>
   <receiver>ProbeSettingSlctvtBscPage</receiver>
   <slot>onDefault()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_showAdv</sender>
   <signal>clicked()</signal>
   <receiver>ProbeSettingSlctvtBscPage</receiver>
   <slot>onShowAdv()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_target</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ProbeSettingSlctvtBscPage</receiver>
   <slot>onScrollTar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_nontarget</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ProbeSettingSlctvtBscPage</receiver>
   <slot>onScrollNontar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onScrollTar()</slot>
  <slot>onScrollNontar()</slot>
  <slot>onDefault()</slot>
  <slot>onShowAdv()</slot>
 </slots>
</ui>
